Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -379,6 +415,26 @@ def replace_substring(data: dict | str, original: str, new: str) -> str | dict:
     return data
 
 
+def determine_assignee_filter_field(assignee: str) -> str:
```

Comment:
This should be a comma separated value but it looks like this function takes care for a single value.
Lets assume that we will get either pretty list or email list.